python/graphstorm/trainer/ep_trainer.py

```diff
@@ -213,11 +214,14 @@ def fit(self, train_loader, num_epochs,
                 break
 
         rt_profiler.save_profile()
-        print("Peak Mem alloc: {:.4f} MB".format(th.cuda.max_memory_allocated(device) / 1024 /1024))
+        print("Peak GPU Mem alloc: {:.4f} MB".format(th.cuda.max_memory_allocated(device) /
```

Comment:
We need to check whether cuda device is avaliable before we collect GPU Mem alloc info.

Reply:
I have changed it here so that only one info is printed depending on if cuda is used or not. But for the log_params I left both parameters because max GPU memory will be just 0 in case of calculations on CPU and all the runs will have consistent  parameters.